src/EnergyPlus/IOFiles.cc

```diff
@@ -613,9 +614,15 @@ class custom_arg_formatter : public arg_formatter
                 // 0 pad the end
                 specs()->alt = true;
 
-                if (specs()->precision > 0) {
+                bool initialPrecisionWas1 = false;
+                if (specs()->precision > 1) {
                     // reduce the precision to get rounding behavior
                     --specs()->precision;
+                } else {
+                    // We need AT LEAST one in precision so we capture a '.' below
+                    initialPrecisionWas1 = true;
+                    specs()->precision = 1;
+                    ++specs()->width;
```

Comment:
Another path: if requested precision is 1 (which can be understandable perhaps) or (0 which makes no sense...)